src/main/java/pl/joegreen/sergeants/simulator/Replay.java

```diff
@@ -0,0 +1,77 @@
+package pl.joegreen.sergeants.simulator;
+
+class Replay {
```

Comment:
This is half done. This is now only used to generate a map from http://dev.generals.io/replays.
When fully implemented it should be called Replay and moved to the not yet implemented replay functionality.